modules/nf-core/quartonotebook/tests/with-parametrization.config

```diff
@@ -0,0 +1,10 @@
+env {
+    XDG_CACHE_HOME = "./.xdg_cache_home"
+    XDG_DATA_HOME = "./.xdg_data_home"
```

Comment:
Wouldn't it make sense for this to be set by default, i.e. add it to the `script` section of the process?

Reply:
Good point! I have update the module with this suggestion.